### PR TITLE
feature/137 text changes

### PR DESCRIPTION
**Pull request by cschwinderg**
## Related Issues:
* #137 

## Main Changes:
* Made text changes per Christopher Clipper's suggestions.
  * I skipped over the twitter/x item (marked out in ticket).

## Steps To Test:
1. Verify text changes match up with those listed in the ticket.

### CODE REVIEW

File: app/app/public/index.html [hunk 1]
```diff
@@ -851,6 +844,11 @@ <h4>
...
                 <p id="conclusion2" style="font-weight: bold"></p>
               </div>
+
+              <p>
+                For questions or comments, email EPA’s CGP staff at
+                <a href="mailto:cgp@epa.gov">cgp@epa.gov</a>.
+              </p>
             </div>
           </article>
         </div>
```
Comment (maxdiebold-erg) [inline app/app/public/index.html:854]: This is fine as is, but another possibility that I've seen on other EPA pages (e.g. epa.gov) is to move this into the `l-page_footer`, usually paired with the "last updated" snippet:
```suggestion
            </div>
          </article>
        </div>
        <div class="l-page__footer">
          <div class="l-constrain">
            <div class="l-page__footer-contact">
              For questions or comments, email EPA’s CGP staff at
              <a href="mailto:cgp@epa.gov">cgp@epa.gov</a>.
            </div>
            <div class="l-page__footer-last-updated">
              Last updated on January 4, 2024
            </div>
          </div>
        </div>
```
In this case, the `p` element with the `footer__last-updated` class would be removed below (lines 1111-1113) and the `has-footer` class would be added to the `div` on line 558.

**Review by maxdiebold-erg — CHANGES_REQUESTED** [review]
The text changes look good!

I made one suggestion concerning the email link, feel free to take it or leave it.

Also, unrelated to this issue, but to save on a separate PR, I noticed that the step indicator circles are looking a bit oblong depending on the amount of text in the step. This could be avoided with this addition to the style overrides:
```
      .usa-step-indicator__current-step {
        flex-shrink: 0;
      }
```

File: app/app/public/index.html (hunk 1 above)
Comment (cschwinderg) [reply app/app/public/index.html:854]: It doesn't match up with pattern lab template, but it definitely looks better so I'm going to change it.

Hopefully one day EPA will have a standard template that is actually used by epa.gov.

**Review by maxdiebold-erg — APPROVED** [review]

**Comment by maxdiebold-erg** [comment]
Thanks for adding in those changes!

**Pull request merged**